apps/ejabberd/src/mod_mam_muc_odbc_async_pool_writer.erl

```diff
@@ -43,7 +43,7 @@
                 max_packet_size     :: non_neg_integer(),
                 max_subscribers     :: non_neg_integer(),
                 host                :: ejabberd:server(),
-                conn,
+                connection_pool     :: atom(),
```

Comment:
the point of having a separate connection here is not to wait for checking-out pool. And not to block the default pool.